source/includes/_portfolio.md

```diff
@@ -0,0 +1,265 @@
+    },
+    {
+        "slug": "scheme name",
+        "name": "Schema name",
+        "updated_at": "2021-03-04T10:09:54.671Z"
+    },
+    {
+        "slug": "country",
+        "name": "Country (ISO3166)",
+        "updated_at": "2019-04-11T15:30:36.132Z"
+    },
+    // ... trimmed for brevity ...
+] 
+```
+
+## GET /lists/:slug
+Requests info on a specific picklist and lists all the available picklist items.
+
+```shell
+curl https://api.gresb.com/api/v0/lists/:slug \
+```
+
+> Response: 
+
+```json
+{
+    "slug": "country",
+    "name": "Country (ISO3166)",
+    "updated_at": "2019-04-11T15:30:36.132Z",
+    "items": [
+        {
+            "label": "Afghanistan",
+            "value": "AF"
+        },
+        {
+            "label": "Albania",
+            "value": "AL"
+        },
+        {
+            "label": "Algeria",
+            "value": "DZ"
+        },
+    	// ... trimmed for brevity ...
+	]
+}
+```
+
+## GET /entities/{entity_id}/responses
+Requests the assessment responses associated with the entity.
+
+```shell
+curl https://api.gresb.com/api/v0/entities/{entity_id}/responses \
+  -H "Authorization: Bearer $ACCESS_TOKEN"
+```
+
+> Response:
+
+```json
+[
+    {
+        "id": {response_id},
+        "entity_id": {entity_id},
+        "business_line": "real_estate",
+        "survey_year": {Survey Year},
+        "name": "API Test Entity",
+        "manager": "GRESB",
+        "status": "in_progress",
+        "survey_open": true,
+        "assessment_type": "real_estate",
+        "percent_completed": 65,
+        "created_at": "2022-11-29T11:39:12.267Z",
+        "updated_at": "2022-12-07T08:53:43.468Z"
+    },
+    {
+        "id": {response_id},
+        "entity_id": {entity_id},
+        "business_line": "real_estate",
+        "survey_year": {Survey Year},
+        "name": "API Test Entity",
+        "manager": "GRESB",
+        "status": "new",
+        "survey_open": true,
+        "assessment_type": "sfdr_re",
+        "percent_completed": 0,
+        "created_at": "2022-12-08T08:48:18.399Z",
+        "updated_at": "2022-12-08T08:48:20.967Z"
+    }
+]
+```
+
+## GET /entities/{entity_id}/responses/{response_id}
+Requests the data on the assessment response.
+
+```shell
+curl https://api.gresb.com/api/v0/entities/{entity_id}/responses/{response_id} \
+  -H "Authorization: Bearer $ACCESS_TOKEN"
+```
+
+> Response:
+
+```json
+[
+    {
+        "id": {response_id},
+        "entity_id": {entity_id},
+        "business_line": "real_estate",
+        "survey_year": {Survey Year},
+        "name": "API Test Entity",
+        "manager": "GRESB",
+        "status": "in_progress",
+        "survey_open": true,
+        "assessment_type": "real_estate",
+        "percent_completed": 65,
+        "created_at": "2022-11-29T11:39:12.267Z",
+        "updated_at": "2022-12-07T08:53:43.468Z"
+    }
+]
+```
+
+## GET /entities/{entity_id}/responses/{response_id}/values **PLACEHOLDER**
```

Comment:
Probably won't be implemented given that we're short on time - we'll work with a slimmed down version of the codebook instead.

Reply:
I removed this section.